.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 ./package-lock.json
 .idea/
+./config.js
```

Comment:
Maybe we should create a template? like config.js.template? and ignore the other one

Reply:
I can create a script for creating config.js file

Reply:
So I have to make a PR until it is resolved.

Reply:
I'll merge it :) But better send that script :))